cluster-autoscaler/core/scale_up.go

```diff
@@ -106,7 +106,7 @@ func ScaleUp(context *AutoscalingContext, unschedulablePods []*apiv1.Pod, nodes
 		}
 
 		for _, pod := range unschedulablePods {
-			err = context.PredicateChecker.CheckPredicates(pod, nodeInfo)
+			err = context.PredicateChecker.CheckPredicates(pod, nodeInfo, true)
```

Comment:
Instead of explicit booleans can you have constants that are more descriptive (and perhaps even declare a specific type to make it more robust)?